src/citrine/resources/design_workflow.py

```diff
@@ -49,10 +50,16 @@ def restore(self, workflow_id: UUID):
 
         """
         url = self._path_template.format(project_id=self.project_id) \
-            + "/{}/restore".format(workflow_id)
+              + "/{}/restore".format(workflow_id)
         self.session.put_resource(url, {})
 
     def delete(self, uid: Union[UUID, str]) -> Response:
         """Design Workflows cannot be deleted; they can be archived instead."""
         raise NotImplementedError(
             "Design Workflows cannot be deleted; they can be archived instead.")
+
+    def list_archived(self,
+                      page: Optional[int] = None,
```

Comment:
Added **kwargs to list_archived() as a last argument.

Reply:
Sorry I wasn't clear, I didn't mean add `**kwargs` to the method definition (that would allow users to specify any `kwarg` they wanted when calling the method, but since we don't then use the `kwargs` in the method body, it would make no difference)

I meant make the existing arguments require keywords. So instead of the method definition being `def list_archived(self, page: Optional[int] = None, ...`, use the `*` to force users to use keyword arguments when calling the method. Like so: `def list_archived(self, *, page: Optional[int] = None, ...`. Notice the `*` right after the `self` argument.

In the second case, the method must be called like so: `collection.list_archived(page=page, per_page=per_page` instead of `collection.list_archived(page, per_page` without that `*`.

Reply:
Fixed this one.